### ISSUE TITLE
Unable to create a vpcaccess Connector without a name

### ISSUE THREAD

**Issue by ben-sellar** [pulumi/pulumi-google-native#358]
## Hello!

- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already)

## Issue details

Connectors don't have names, they just have a connectorId, however I'm forced to pass both, which then fails to deploy.

Trying to add a connector to give a cloud function access to a VPC.

### Steps to reproduce

```typescript 
import { Connector } from "@pulumi/google-native/vpcaccess/v1";

const network = new Network("vpc", {
  autoCreateSubnetworks: false,

  routingConfig: {
    routingMode: "REGIONAL",
  },
});

const connector = new Connector("functions-connector", {
  network: network.selfLink,
});

```

Expected: Connection to be created, name (functions-connector) gets passed as ContainerId, *NOT* name,
Actual: typescript forces me to add a connectionId (`Property 'connectorId' is missing in type '{ network: Output<string>; }' but required in type 'ConnectorArgs'`), both ConnectionId *and* name get passed, get back 

```google-native:vpcaccess/v1:Connector (functions-connector):
    error: error sending request: googleapi: Error 400: Connector name must not be set.: "https://vpcaccess.googleapis.com/v1/projects/projectName(((/locations/***/connectors?connectorId=functions-connector" map[connectorId:functions-connector location:*** name:projects/***/locations/***/connectors/functions-connector-*** network:https://www.googleapis.com/compute/v1/projects/***/global/networks/vpc-*** project:***]
```

**Comment by ben-sellar** [comment]
Reproduced in Go too

```go
connector, err := vpcaccess.NewConnector(ctx, "connector", &vpcaccess.ConnectorArgs{
			ConnectorId: pulumi.String("here-is-my-id"),
			Location:    pulumi.String("***"),
			Project:     pulumi.String("***"),
		})
```

**Comment by ProfXponent** [comment]
bumping as it still seems to be an issue with the node sdk for google-native

**Comment by viveklak** [comment]
Fixed in #618.

Sample code that works with the above fix:
```
import {Connector} from "@pulumi/google-native/vpcaccess/v1";
import {Network} from "@pulumi/google-native/compute/v1";

const network = new Network("vpc", {
  autoCreateSubnetworks: false,

  routingConfig: {
    routingMode: "REGIONAL",
  },
});

const connector = new Connector("functions-connector", {
  connectorId: "connector",
  network: network.name,
  ipCidrRange: "10.132.0.0/28",
  minInstances: 2,
  maxInstances: 5,
});
```
